Fix default and stage of AntreaProxy, AntreaPolicy and Traceflow in feature-gates.md

yktsubo opened the pull request:
Fix default and stage of AntreaProxy, AntreaPolicy and Traceflow in feature-gates.md
https://github.com/vmware-tanzu/antrea/releases/tag/v0.11.0

Signed-off-by: Yuki Tsuboi <ytsuboi@vmware.com>

antoninbas reviewed (commented):
Could you rebase your PR and resolve the conflicts? Some of these discrepancies were addressed in a PR that just got merged.
Please also consider changing the `Stage` column for `AntreaProxy`, `AntreaPolicy` and `Traceflow` to `Beta`. Thanks!

yktsubo commented:
> Could you rebase your PR and resolve the conflicts? Some of these discrepancies were addressed in a PR that just got merged.
> Please also consider changing the `Stage` column for `AntreaProxy`, `AntreaPolicy` and `Traceflow` to `Beta`. Thanks!
Sorry, I wasn't aware of that change. I addressed your comment. Thanks.

antoninbas commented:
/skip-all
